plugins/indexing/batching/module.go

```diff
@@ -31,19 +32,24 @@ func ExtractUpdate(ctx *types.BlockContext, cdc codec.Codec, logger *log.Logger,
 		}
 
 		data := struct {
-			ID             string `json:"result_id"`
-			DrID           string `json:"dr_id"`
-			Version        string `json:"version"`
-			BlockHeight    string `json:"block_height"`
-			ExitCode       uint32 `json:"exit_code"`
-			GasUsed        string `json:"gas_used"`
-			Result         []byte `json:"result"`
-			PaybackAddress string `json:"payback_address"`
-			SedaPayload    string `json:"seda_payload"`
-			Consensus      bool   `json:"consensus"`
+			ID             string    `json:"result_id"`
+			DrID           string    `json:"dr_id"`
+			DrBlockHeight  string    `json:"dr_block_height"`
+			Timestamp      time.Time `json:"timestamp"`
+			Version        string    `json:"version"`
+			BlockHeight    string    `json:"block_height"`
+			ExitCode       uint32    `json:"exit_code"`
+			GasUsed        string    `json:"gas_used"`
+			Result         []byte    `json:"result"`
+			PaybackAddress string    `json:"payback_address"`
+			SedaPayload    string    `json:"seda_payload"`
+			Consensus      bool      `json:"consensus"`
```

Comment:
Not important, but we are following this order in other parts of the project.

```suggestion
			ID             string    `json:"result_id"`
			DrID           string    `json:"dr_id"`
			DrBlockHeight  string    `json:"dr_block_height"`
			Version        string    `json:"version"`
			BlockHeight    string    `json:"block_height"`
			Timestamp      time.Time `json:"timestamp"`
			ExitCode       uint32    `json:"exit_code"`
			GasUsed        string    `json:"gas_used"`
			Result         []byte    `json:"result"`
			PaybackAddress string    `json:"payback_address"`
			SedaPayload    string    `json:"seda_payload"`
			Consensus      bool      `json:"consensus"`
```